### PR TITLE
add Anthony's github and twitter links and a link to GHW2016 tutorials

### PR DESCRIPTION
**Pull request by lsetiawan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,8 @@ <h3 class="to-animate">
 					<h2 class="to-animate">SCHEDULE</h2>
 					<div class="row">
 						<div class="col-md-8 col-md-offset-2 subtext to-animate">
-							<h3>Schedule TBD</h3>
+							<h3>Schedule TBD.</h3>
+                            <h3>Click <a href="https://geohackweek.github.io/ghw2016/">here</a> to view the schedule of tutorials from our 2016 Geo Hackweek.</h3>
```
Comment (lsetiawan) [inline index.html:171]: @aaarendt Why do we need this?

File: index.html [hunk 2]
```diff
@@ -273,14 +274,13 @@ <h3 class="to-animate">List of Our Awesome Instructors</h3>
...
+                                 Research Scientist, <a href="http://escience.washington.edu/people/anthony-arendt/">eScience Institute</a> and
+                                <a href="http://psc.apl.uw.edu/people/investigators/anthony-arendt/"> Applied Physics Lab</a>
                             </span>
-							<!--<ul class="social-media">-->
-								<!--<li><a href="#" class="twitter"><i class="icon-twitter"></i></a></li>-->
-								<!--<li><a href="#" class="github"><i class="icon-github-alt"></i></a></li>-->
-							<!--</ul>-->
+							<ul class="social-media">
+								<li><a href="https://twitter.com/aaarendt" class="twitter"><i class="icon-twitter"></i></a></li>
+								<li><a href="https://github.com/aaarendt" class="github"><i class="icon-github-alt"></i></a></li>
```
Comment (lsetiawan) [inline index.html:282]: looks good here!

File: index.html (hunk 1 above)
Comment (geohackweek) [reply index.html:171]: we don't currently have any link to 2016 material. I'm thinking people will want to see that in deciding whether to apply.

Comment (lsetiawan) [reply index.html:171]: Also, that doesn't seem like a schedule anymore from 2016, but rather a list of previous tutorials. So maybe reword to view last year's tutorials?
